File: CS/WebSite/App_Code/MyCustomDictionary.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using DevExpress.XtraSpellChecker;
using System.Globalization;

public class MyCustomDictionary : SpellCheckerCustomDictionary {
    public override bool Loaded { get { return true; } }
}

File: VB/WebSite/App_Code/MyCustomDictionary.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports DevExpress.XtraSpellChecker
Imports System.Globalization

Public Class MyCustomDictionary
	Inherits SpellCheckerCustomDictionary
	Public Overrides ReadOnly Property Loaded() As Boolean
		Get
			Return True
		End Get
	End Property
End Class